src/domain/users-ip/index.types.d.ts

```diff
@@ -0,0 +1,15 @@
+type UserIp = {
```

Comment:
Not sure I like the name - how about `UserConnectionsRecord` or at least it the `Ip` should probably be plural `UserIPs` right?

Reply:
moving to `UserIPs` for now